components/content/space-game/3d/Enemies.vue

```diff
@@ -0,0 +1,42 @@
+<script setup lang="ts">
+import * as THREE from 'three'
```

Comment:
Thanks for having a look.

> import * add three to the final bundle

Last I read, `import * from [...]` is tree-shook. I tried to verify it by building, but after ...

`pnpm build`

... and ...

`pnpm generate`

... I'm getting ...

```
[4:33:13 PM]  ERROR  Nuxt Build Error: [vite]: Rollup failed to resolve import "#components" from "virtual:nuxt:/oss/tres/tres_lab/.nuxt/mdc-image-component.mjs".
This is most likely unintended because it can break your application at runtime.
If you do want to externalize this module explicitly add it to
build.rollupOptions.external
```

Any idea how to fix?